### PR TITLE
dev => master (nov 6th, 2015)

### PR DESCRIPTION
**Pull request by mvoropaiev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+### Produce a message:  
+Code:
+```py
+from flask import Flask
+from flask.ext.rabbitplay import Rabbit
+
+app = Flask('rabbit_example')
+
+rabbit = Rabbit(app)
+with app.app_context():
```
Comment (f0t0n) [inline README.md]: let's use an example from `examples/flask_produce.py`

File: flask_rabbitplay.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        app.extensions['rabbitplay'] = self
+
+        if hasattr(app, 'teardown_appcontext'):
+            app.teardown_appcontext(self.teardown)
+        else:
+            app.teardown_request(self.teardown)
+
+    def connect(self):
+        self.rabbit = Connection.instance(
```
Comment (f0t0n) [inline flask_rabbitplay.py]: just `return Connection.instance(...)`

File: flask_rabbitplay.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        return self.rabbit
+
+    def teardown(self, exception):
+        ctx = stack.top
+        if hasattr(ctx, 'rabbitplay_conn'):
+            ctx.rabbitplay_conn.close()
+
+    @property
+    def connection(self):
+        ctx = stack.top
```
Comment (f0t0n) [inline flask_rabbitplay.py:69]: just 

```
ctx.rabbitplay_conn = self.connect()
return ctx.rabbitplay_conn
```

File: flask_rabbitplay.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        if ctx is not None:
+            if not hasattr(ctx, 'rabbitplay_conn'):
+                ctx.rabbitplay_conn = self.connect()
+            return ctx.rabbitplay_conn
+
+    def produce(self, queue, message):
+        producer = Producer(self.connection)
```
Comment (f0t0n) [inline flask_rabbitplay.py]: create `self.producer ` in the class constructor.

**Comment by mvoropaiev** [comment]
@f0t0n, have made some changes according to your feedback, please review, thanks!

**Comment by mvoropaiev** [comment]
Closes #1

**Pull request merged**